src/lib/draw-and-pick.js

```diff
@@ -196,6 +196,15 @@ export function pickLayers(gl, {
     if (info) {
       infos.set(info.layer.id, info);
     }
+
+    // update pickingSelectedColor so picked model is highlighted.
+    if (layer.props.pickingHighlightColor !== NULL_PICKING_COLOR) {
```

Comment:
This seems wrong - I assume that the highlight color is the color that should be used to render (highlight) a selected item, and this it is a real color (has nothing to do with picking colors). Why not simply use `null` to detect whether no highlighting is desired.